cla_backend/templates/admin/reports/base.html

```diff
@@ -30,6 +30,7 @@
   {{ form.media }}
 
   <script id="reportTasks">
+    const csrfToken = "{{ csrf_token }}";
```

Comment:
Whilst this is true, we are confident that the `csrf_token` is passed directly from Django.

This can be mitigated, as semgrep points out, by adding it to the page as a hidden element if desired.

Reply:
Anyone can view the CSRF token in the session storage so this is not a risk